src/ontology/mondo-edit.obo

```diff
@@ -296546,7 +296546,7 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/2578/guanidi
 [Term]
 id: MONDO:0013000
 name: porphyria due to ALA dehydratase deficiency
-def: "Porphyria of doss or deficiency of delta-aminolevulinic acid dehydratase (DALAD) is an extremely rare form of acute hepatic porphyria characterized by neuro-visceral attacks without cutaneous manifestations." [Orphanet:100924]
+def: "ALAD-related porphyria is an extremely rare form of acute hepatic porphyria characterized by neuro-visceral attacks without cutaneous manifestations." [https://clinicalgenome.org/affiliation/40097/, https://orcid.org/0000-0002-0587-4693]
```

Comment:
We do not (or try not to) repeat the term label in the definition. Therefore the definition should be:
"an extremely rare form of..."